From c6b36901b8a3771b8c3762faa67f08fc5611de66 Mon Sep 17 00:00:00 2001
From: Anita Zhang <the.anitazha@gmail.com>
Date: Mon, 14 Jun 2021 10:48:01 -0700
Subject: [PATCH] Revert "core: prevent excessive /proc/self/mountinfo parsing"

This reverts commit d586f642fd90e3bb378f7b6d3e3a64a753e51756.

sd-event rate limiting has a few issues in 248 that need to be worked
out. See: https://github.com/systemd/systemd/pull/19811 and
https://github.com/systemd/systemd/pull/19924. Let's try again in 249.
---
 src/core/mount.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/core/mount.c b/src/core/mount.c
index ca5d0939a1..2939062161 100644
--- a/src/core/mount.c
+++ b/src/core/mount.c
@@ -1859,12 +1859,6 @@ static void mount_enumerate(Manager *m) {
                         goto fail;
                 }
 
-                r = sd_event_source_set_ratelimit(m->mount_event_source, 1 * USEC_PER_SEC, 5);
-                if (r < 0) {
-                        log_error_errno(r, "Failed to enable rate limit for mount events: %m");
-                        goto fail;
-                }
-
                 (void) sd_event_source_set_description(m->mount_event_source, "mount-monitor-dispatch");
         }
 
-- 
2.31.1

